File: linmour-order-module-go/server/models/OrderItem.go
package models

// OrderItem 代表订单条目信息的结构体
type OrderItem struct {
	ID         string  `json:"id" gorm:"column:id"`                  // 订单条目ID
	Name       string  `json:"name" gorm:"column:name"`              // 商品名字
	ShopId     string  `json:"shopId" gorm:"column:shop_id"`         // 商店ID
	Status     int16   `json:"status" gorm:"column:status"`          // 状态
	ProductID  int64   `json:"productID" gorm:"column:product_id"`   // 商品ID
	SortID     string  `json:"sortID" gorm:"column:sort_id"`         // 分类ID
	Picture    string  `json:"picture" gorm:"column:picture"`        // 商品图
	Price      float64 `json:"price" gorm:"column:price"`            // 价格
	Sort       string  `json:"sort" gorm:"column:sort"`              // 分类
	PropsText  string  `json:"propsText" gorm:"column:props_text"`   // 规格文本
	Quantity   int16   `json:"quantity" gorm:"column:quantity"`      // 数量
	OrderID    string  `json:"orderID" gorm:"column:order_id"`       // 订单ID
	CreateTime string  `json:"createTime" gorm:"column:create_time"` // 创建时间
	UpdateTime string  `json:"updateTime" gorm:"column:update_time"` // 更新时间
	CreateBy   string  `json:"createBy" gorm:"column:create_by"`     // 创建人
	UpdateBy   string  `json:"updateBy" gorm:"column:update_by"`     // 更新人
	Deleted    int     `json:"deleted" gorm:"column:deleted"`        // 是否删除
}

File: linmour-order-module/linmour-order-api/src/main/java/com/linmour/order/pojo/Do/OrderItem.java
package com.linmour.order.pojo.Do;

import com.baomidou.mybatisplus.annotation.*;
import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
import com.linmour.product.pojo.Dto.ProductSortAndOption;
import lombok.Data;

import java.io.Serializable;
import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.List;

/**
 * @TableName order_item
 */
@TableName(value = "order_item")
@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class OrderItem implements Serializable {
    /**
     *
     */

    private Long id;

    /**
     * 商品名字
     */

    private String name;

    /**
     *
     */

    private Long shopId;
    private Integer status;

    private Long productId;

    /**
     * 分类id
     */

    private Long sortId;

    /**
     * 商品图
     */

    private String picture;

    /**
     * 如果商品有规格就显示价格最低的配置
     */

    private BigDecimal price;
    private String sort;

    private String propsText;

    private Integer quantity;
    private String orderId;


    /**
     *
     */
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    @TableField(value = "create_time", fill = FieldFill.INSERT)
    private LocalDateTime createTime;

    /**
     *
     */
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    @TableField(value = "update_time", fill = FieldFill.INSERT_UPDATE)
    private LocalDateTime updateTime;

    /**
     *
     */
    @TableField(value = "create_by", fill = FieldFill.INSERT)
    private Long createBy;

    /**
     *
     */
    @TableField(value = "update_by", fill = FieldFill.INSERT_UPDATE)
    private Long updateBy;

    /**
     *
     */
    @TableField(value = "deleted")
    private Integer deleted;

    @TableField(exist = false)
    private static final long serialVersionUID = 1L;
}